cPP_DSC.adoc

```diff
@@ -333,7 +333,11 @@ One way to categorize keys is by the cryptographic functions they are allowed to
 
 ==== Sessions
 
-Users may use their keys multiple times while in the DSC. Because authorization using public key methods tends to be resource intensive (i.e. uses a fair amount of internal memory and takes a long time), the DSC can use sessions to enforce authorization and manage access to the key within it. As an alternative to requiring authorization for each access to a key, the DSC could allow the user or owner of the key to open a session and provide the authorization when being used for the first time, then maintain the session and authorization using a series of less resource-intensive challenges and responses. Alternatively, in some instances, the DSC may require additional authorization (such as an elevation of privileges) to access keys (or different, related keys). Such a protocol of challenges and responses may generate and use ephemeral authorization tokens, which would be one form of critical security parameter (CSP). The DSC may have to switch session contexts in and out of the DSC to external temporary storage, which necessitates the protection of these CSPs. Such a session context is one type of SDO, to be discussed later.
+For a DSC, a connection is established between a CA and the services provided by the DSC when keys or services are requested. Each time the CA establishes a connection to the DSC the CA is authenticated to ensure the CA has authorization to the requested keys. Since a CA (acting as an agent of the user) may utilize their DSC keys multiple times, the establishment of individual connections for each use can be a resource constraint for the DSC as authorization methods using public keys tend to be resource intensive (i.e. uses a fair amount of internal memory and takes a long time).
```

Comment:
The CA, as defined in this cPP is Client Application. I think though, that CA should become CApp so it removes that ambiguity (but I would make that as a separate change.

I added this otherwise to try and actually remove the idea that the DSC may communicate to something larger (i.e. that it only communicates directly on device, and anything else is the purview of such a client application).

@smuellerDD With the idea that CA is not a certificate authority, would this be acceptable (and I will create a separate issue to change CA to CApp)?